docs: Broken links found

# Summary

| Status        | Count |
|---------------|-------|
| 🔍 Total      | 699   |
| ✅ Successful | 681   |
| ⏳ Timeouts   | 0     |
| 🔀 Redirected | 0     |
| 👻 Excluded   | 14    |
| ❓ Unknown    | 0     |
| 🚫 Errors     | 4     |

## Errors per input

### Errors in conferences/2025/data.json

* [404] [https://lu.ma/j02yktcu](https://lu.ma/j02yktcu) | Network error: Not Found

### Errors in conferences/2025/identity.json

* [403] [https://internetidentityworkshop.com/](https://internetidentityworkshop.com/) | Network error: Forbidden

### Errors in conferences/2025/general.json

* [ERROR] [https://saltmarch.com/code-of-conduct](https://saltmarch.com/code-of-conduct) | Network error: error sending request for url (https://saltmarch.com/code-of-conduct) Maybe a certificate error?
* [404] [https://slashnew.tech/cfp](https://slashnew.tech/cfp) | Network error: Not Found
[Full Github Actions output](https://github.com/tech-conferences/conference-data/actions/runs/13341084964?check_suite_focus=true)

--- a/conferences/2023/general.json
+++ b/conferences/2023/general.json
@@ -429,7 +429,6 @@
     "country": "India",
     "online": false,
     "locales": "EN",
-    "cocUrl": "https://saltmarch.com/code-of-conduct",
     "twitter": "@developersummit"
   },
   {

--- a/conferences/2024/general.json
+++ b/conferences/2024/general.json
@@ -269,7 +269,6 @@
     "country": "India",
     "online": false,
     "locales": "EN",
-    "cocUrl": "https://saltmarch.com/code-of-conduct",
     "twitter": "@developersummit"
   },
   {

--- a/conferences/2025/data.json
+++ b/conferences/2025/data.json
@@ -165,8 +165,6 @@
     "online": false,
     "locales": "EN",
     "cocUrl": "https://shift.infobip.com/us/code/",
-    "cfpUrl": "https://lu.ma/j02yktcu",
-    "cfpEndDate": "2025-03-01",
     "twitter": "@InfobipShift"
   },
   {

--- a/conferences/2025/general.json
+++ b/conferences/2025/general.json
@@ -368,7 +368,6 @@
     "country": "India",
     "online": false,
     "locales": "EN",
-    "cocUrl": "https://saltmarch.com/code-of-conduct",
     "cfpUrl": "https://developersummit.com/cfp",
     "cfpEndDate": "2024-09-30",
     "twitter": "@developersummit"
@@ -447,9 +446,7 @@
     "country": "Australia",
     "online": false,
     "locales": "EN",
-    "cocUrl": "https://slashnew.tech/coc/",
-    "cfpUrl": "https://slashnew.tech/cfp",
-    "cfpEndDate": "2025-01-31"
+    "cocUrl": "https://slashnew.tech/coc/"
   },
   {
     "name": "DevConf Cape Town",

